source/SpinalHDL/Structuring/clock_domain.rst

```diff
@@ -9,7 +9,7 @@ Clock domains
 Introduction
 ------------
 
-In *SpinalHDL*\ , clock and reset signals can be combined to create a **clock domain**. Clock domains can be applied to some areas of the design and then all synchronous elements instantiated into those areas will then **implicitly** use this clock domain.
+In SpinalHDL, clock and reset signals can be combined to create a **clock domain**. Clock domains can be applied to some areas of the design and then all synchronous elements instantiated into those areas will then `implicitly` use this clock domain.
```

Comment:
Why use inline formatting instead of the bold text?
(I prefer the bold text, but I don't have a strong opinion about it)

```suggestion
In SpinalHDL, clock and reset signals can be combined to create a **clock domain**. Clock domains can be applied to some areas of the design and then all synchronous elements instantiated into those areas will then **implicitly** use this clock domain.
```

Reply:
I think I axed the bolds here because they looked inconsistent with the majority of the other pages I've edited. That said, I actually *like* the bold formatting, so I'll happily throw that back in, since I think it makes the docs easier to read.